main.go

```diff
@@ -1,32 +1,69 @@
 package main
 
-import (
+import(
 	"fmt"
 	"log"
 	"net/http"
-
 	"github.com/EaseApp/web-backend/src/db"
-
 	"github.com/codegangsta/negroni"
 	"github.com/gorilla/mux"
+	user "github.com/EaseApp/web-backend/src/app/controllers/user"
+	application "github.com/EaseApp/web-backend/src/app/controllers/application"
+	dao "github.com/EaseApp/web-backend/src/app/dao"
 )
 
+
+
+func NewStaticUserHandler(w http.ResponseWriter, req *http.Request){
+	_ = user.InsertStaticUser()
+	http.Redirect(w, req, "/users", http.StatusFound)
+}
+
+func DBCountHandler(w http.ResponseWriter, req *http.Request){
+	vars := mux.Vars(req)
+	db := vars["db"]
+
+	// Move RecordCount out of user DAO and into generic DAO
+	fmt.Fprintf(w, "Db (%v) has (%v) objects.", db, dao.RecordCount(db))
+}
+
 func main() {
 	log.Println("Starting server...")
 
-	router := mux.NewRouter()
-	router.HandleFunc("/", func(w http.ResponseWriter, req *http.Request) {
-		fmt.Fprintf(w, "Welcome to the home page!")
-	})
-
 	// TODO: Use command line flag credentials.
 	client, err := db.NewClient("localhost:28015")
 	if err != nil {
 		log.Fatal("Couldn't initialize database: ", err.Error())
 	}
+	db.CreateEaseDb(client)
+	db.CreateUserTable(client)
+	db.CreateDbTable(client)
+
+	dao.Init(client.Session)
+	user.Init(client.Session)
+	application.Init(client.Session)
+
+	router := mux.NewRouter()
+	router.HandleFunc("/", func(w http.ResponseWriter, req *http.Request) {
+		fmt.Fprintf(w, "Welcome to the home page!")
+	})
+
+
+	router.HandleFunc("/static/user/new", NewStaticUserHandler)
+
+	// These should be POST, but it's easier to test with GET
+	router.HandleFunc("/users/sign_in", user.SignInHandler)
+	router.HandleFunc("/users/sign_up", user.SignUpHandler)
+
+	router.HandleFunc("/count/{db}", DBCountHandler)
+	router.HandleFunc("/{db}", user.FetchAllHandler)
+
+	router.HandleFunc("/{client}/{application}", application.QueryApplicationHandler).Methods("POST")
+	router.HandleFunc("/{client}/{application}/new", application.CreateApplicationHandler).Methods("POST")
+
 	defer client.Close()
 
 	n := negroni.Classic()
 	n.UseHandler(router)
-	n.Run(":3001")
+	n.Run(":3000")
```

Comment:
It was either change this line or change the vagrant file and reload it, this seemed easier. The vagrant file is only port-forwarded for 3000